onadata/settings/docker_read_replica.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding=utf-8 -*-
+"""
+Example local_settings.py used by the Dockerfile.
+"""
+# this system uses structured settings.py as defined in
+# http://www.slideshare.net/jacobian/the-best-and-worst-of-django
+#
+# this third-level staging file overrides some definitions in staging.py
+# You may wish to alter it to agree with your local environment
+#
+
+# get most settings from staging_example.py (which in turn, imports from
+# settings.py)
+import os
+import subprocess
+import sys
+
+from onadata.settings.common import *  # noqa
+
+# # # now override the settings which came from staging # # # #
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.contrib.gis.db.backends.postgis',
+        'NAME': 'onadata',
+        'USER': 'onadata',
+        'PASSWORD': 'onadata',
+        'HOST': 'db_master',
+        'PORT': 5432
+    },
+    'replica': {
+        'ENGINE': 'django.contrib.gis.db.backends.postgis',
+        'NAME': 'onadata',
+        'USER': 'onadata',
+        'PASSWORD': 'onadata',
+        'HOST': 'db_replica',
+        'PORT': 5432
+    }
+}
+
+DATABASE_ROUTERS = []  # turn off second database
+SLAVE_DATABASES = ['replica']
+DATABASE_ROUTERS = ['multidb.PinningMasterSlaveRouter']
+MULTIDB_PINNING_SECONDS = 60
+
+# Make a unique unique key just for testing, and don't share it with anybody.
+SECRET_KEY = '~&nN9d`bxmJL2[$HhYE9qAk=+4P:cf3b'
+
+ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
+
+INTERNAL_IPS = ['127.0.0.1']
+
+DEBUG = True
+CORS_ORIGIN_ALLOW_ALL = True
+CHECK_EXPIRED_TEMP_TOKEN = False
+
+# pylint: disable=simplifiable-if-statement
+if len(sys.argv) >= 2 and (sys.argv[1] == "test" or sys.argv[1] == "test_all"):
+    # This trick works only when we run tests from the command line.
+    TESTING_MODE = True
+else:
+    TESTING_MODE = False
+
+CELERY_BROKER_URL = 'amqp://guest:@queue:5672//'
+CELERY_TASK_ALWAYS_EAGER = False
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_CACHE_BACKEND = 'memory'
+CELERY_BROKER_CONNECTION_MAX_RETRIES = 2
+
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
+if TESTING_MODE:
+    MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'test_media/')  # noqa
+    subprocess.call(["rm", "-r", MEDIA_ROOT])
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6150428)